src/FloatingActionButton/FloatingActionButton.js

```diff
@@ -222,10 +210,8 @@ const FloatingActionButton = React.createClass({
       'icons to FloatingActionButtons.');
   },
 
-  componentWillReceiveProps(nextProps, nextContext) {
-    const newState = {
-      muiTheme: nextContext.muiTheme || this.state.muiTheme,
-    };
+  componentWillReceiveProps(nextProps) {
+    const newState = {};
```

Comment:
Can we fix this to only do the setState call if `if (nextProps.disabled !== this.props.disabled)`?